hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/ContainerClientMetrics.java

```diff
@@ -56,27 +56,27 @@ public final class ContainerClientMetrics {
   private MutableCounterLong totalWriteChunkBytes;
 
   @Metric
-  private MutableRate hsyncSynchronizedWorkNs;
+  private OzoneMutableRate hsyncSynchronizedWorkNs;
```

Comment:
If possible, we should keep the original class names.  Package name should be enough to distinguish between Hadoop and Ozone class.  This would help reduce code change mostly to imports.

Reply:
Good idea! Will fix.